daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -901,7 +901,7 @@ object Main {
     val BadExternalVariable = Value(33)
     val UserDefinedFunctionError = Value(34)
     val UnableToCreateProcessor = Value(35)
-    val LayerExecutionError = Value(36)
+    val LayerExecutionError = Value(36) // TODO: remove if remains unused.
```

Comment:
Maybe fatal errors end up converted into a LayerExecutionError?

Reply:
Ah, it looks like fatal layer errors end up with the BugFound exit code. While fatal layer errors are bugs, they are bugs in the layer implementation. It might be better to execit with LayerExecutionError jsut to make it clear that it's not a bug in daffodil but a bug in a layer.